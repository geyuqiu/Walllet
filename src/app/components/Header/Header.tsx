import React from "react";
import {NavLink} from "react-router-dom";

export const Header = () => {
	const linkClass = "inline-block text-gray-800 hover:text-gray-600 mr-4";

	return (
		<header className="bg-green-700 p-6 min-h-20vh flex items-center justify-between flex-wrap">
			<nav className="block">
				<NavLink to="/">
					<span className={linkClass} data-testid="link__wallet">
						Wallet
					</span>
				</NavLink>
				<NavLink to="/fee">
					<span className={linkClass} data-testid="link__fee">
						Fee
					</span>
				</NavLink>
			</nav>
		</header>
	);
};
