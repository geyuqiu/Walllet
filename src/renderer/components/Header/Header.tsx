import React from "react";
import { NavLink } from "react-router-dom";

export const Header = () => {
	const linkClass = "inline-block text-gray-800 hover:text-gray-600 mr-4";

	return (
		<header className="bg-gray-100 p-6">
			<div className="flex items-center justify-between flex-wrap">
				<nav className="block">
					<NavLink to="/">
						<span className={linkClass}>Wallet</span>
					</NavLink>
					<NavLink to="/fee">
						<span className={linkClass}>Fee</span>
					</NavLink>
				</nav>
			</div>
		</header>
	);
};
