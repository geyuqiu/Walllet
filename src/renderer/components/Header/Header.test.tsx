import { render } from "@testing-library/react";
import React from "react";
import { MemoryRouter } from "react-router-dom";

import { Header } from "./Header";

describe("Header", () => {
	it("should match snapshot", () => {
		const { container } = render(
			<MemoryRouter>
				<Header />
			</MemoryRouter>
		);
		expect(container).toMatchSnapshot();
	});
});
