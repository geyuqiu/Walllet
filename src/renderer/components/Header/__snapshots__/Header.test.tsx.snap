// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match snapshot 1`] = `
<div>
  <header
    class="bg-green-700 p-6 min-h-20vh flex items-center justify-between flex-wrap"
  >
    <nav
      class="block"
    >
      <a
        aria-current="page"
        class="active"
        href="/"
      >
        <span
          class="inline-block text-gray-800 hover:text-gray-600 mr-4"
          data-testid="link__wallet"
        >
          Wallet
        </span>
      </a>
      <a
        href="/fee"
      >
        <span
          class="inline-block text-gray-800 hover:text-gray-600 mr-4"
          data-testid="link__fee"
        >
          Fee
        </span>
      </a>
    </nav>
  </header>
</div>
`;
